### PR TITLE
 Added SLES 15 support for DPDK + Database

### PR DESCRIPTION
**Pull request by rmsivakanth**

### CODE REVIEW

File: Testscripts/Linux/dpdkSetup.sh [hunk 1]
```diff
@@ -96,35 +96,37 @@ function installDPDK ()
...
+		LogMsg "Detected UBUNTU 18.04"
 		srcIp=${2}
 		dstIp=${3}
 		ssh ${1} "until dpkg --force-all --configure -a; sleep 10; do echo 'Trying again...'; done"
 		ssh ${1} "add-apt-repository ppa:canonical-server/dpdk-mlx-tech-preview"
 		ssh ${1} "apt-get update"
 		LogMsg "Configuring ${1} for DPDK test..."
-		ssh ${1} "apt-get install -y gcc wget psmisc tar make dpdk-doc dpdk-dev libdpdk-dev librdmacm-dev librdmacm1 build-essential libnuma-dev libpcap-dev ibverbs-utils"
+		ssh ${1} "apt-get install -y gcc wget psmisc tar make dpdk dpdk-doc dpdk-dev libdpdk-dev librdmacm-dev librdmacm1 build-essential libnuma-dev libpcap-dev ibverbs-utils"
 	elif [[ $DISTRO =~ "Ubuntu 16.04" ]];
```
Comment (lizzha) [inline Testscripts/Linux/dpdkSetup.sh]: It seems the commands for 16.04 are the same as for 18.04. If so, they can be merged in a single "if then" block

File: Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1 [hunk 2]
```diff
@@ -54,9 +54,9 @@ if ($isDeployed)
 		{
 			$DataPath = "SRIOV"
             LogMsg "Getting SRIOV NIC Name."
```
Comment (chvalean) [inline Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1]: indentation needs to be fixed in this code block

File: Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1 [hunk 3]
```diff
@@ -131,9 +130,8 @@ collect_VM_properties
 			WaitFor -seconds 20
 		}
 		$finalStatus = RunLinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort -username "root" -password $password -command "cat /root/state.txt"
-		$uploadResults = $true
 		RemoteCopy -downloadFrom $clientVMData.PublicIP -port $clientVMData.SSHPort -username "root" -password $password -download -downloadTo $LogDir -files "*.csv, *.txt, *.log, *.tar.gz"
-
+		$uploadResults = $true
```
Comment (chvalean) [inline Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1]: this is only used as true.
if for whatever purposes it has a role, better move it at the start of the script for visibility.

File: Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1 [hunk 4]
```diff
@@ -100,14 +100,13 @@ if ($isDeployed)
 			if ( $param -imatch "modes" )
 			{
 				$modes = ($param.Replace("modes=",""))
-			}
+			}
```
Comment (chvalean) [inline Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1]: remove blank space at end of lines

File: Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1 [hunk 5]
```diff
@@ -54,9 +54,9 @@ if ($isDeployed)
 		{
 			$DataPath = "SRIOV"
             LogMsg "Getting SRIOV NIC Name."
-            $clientNicName = (RunLinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort -username "root" -password $password -command "route | grep '^default' | grep -o '[^ ]*$'").Trim()
+            $clientNicName = (RunLinuxCmd -ip $clientVMData.PublicIP -port $clientVMData.SSHPort -username "root" -password $password -command "route | grep '^default' | grep -o '[^ ]*$' 2>&1 | ip route | grep default | tr ' ' '\n' | grep eth").Trim()
```
Comment (chvalean) [inline Testscripts/Windows/VERIFY-DPDK-BUILD-AND-TESTPMD-TEST.ps1]: this network interface detection command seems to always go for eth0, and it is used 4 times in the script.
Maybe put it as a single code block in a variable and use it as such? further changes will apply then in all cases.

**Pull request merged**